lib/Date.php

```diff
@@ -49,11 +49,7 @@ public static function fromYearMonthDayString(string $value): self
 
         [, $year, $month, $day] = $matches;
 
-        return self::fromYearMonthDay(
-            (int) $year,
-            (int) $month,
-            (int) $day,
-        );
+        return self::fromYearMonthDay((int) $year, (int) $month, (int) $day);
```

Comment:
This is causing issues with Infection in #381 - Coverage isn't working well on the multi-line construction and it's thinking this is uncovered.